src/html5/parser.rs

```diff
@@ -2103,10 +2103,9 @@ impl<'stream> Html5Parser<'stream> {
                     return;
                 }
 
+                // Ignore for now.
                 // @TODO: Other stuff
-
-                self.insertion_mode = InsertionMode::AfterBody;
-                self.reprocess_token = true;
+                return;
```

Comment:
This change helps to get one of the unit tests below to pass.

Reply:
This should not happen this way. Reprocess token means the current token is well, reprocessed so it should stay in the loop. I think there is something else not quite ok in the parser (which might take a while to find,.. there sometimes really nasty issues that are very easily overlooked)

Reply:
Ok, no worries.  I can back out this change and add `#[ignore]` to the unit test.

Reply:
I've backed that change out.  Apparently it was an experiment that I tried as I was poking around that wasn't needed.

I'd argue the larger issue is that there's no test (of the `make test` variety) currently protecting it.  The code that was dropped was presumably important, so there should be a test that fails the checks when it's dropped.